### ISSUE TITLE
Backend editing with checkbox menu gives "Too few argument to build the query string" 

### ISSUE THREAD

**Issue by chibko** [alnv/catalog-manager#185]
I have 3 catalogs
- 1 is called "DOCS"
- 1 is "REGION" with a 1 child table "DEP"

In DOCS
- 1 Field "Region" : Checkbox Menu, options from the REGION table, Multiple checked and Submit on change checked
- 1 Field "Dep" : Checkbox Menu, options from the DEP table, Multiple checked and Filter like this : Parent ID IN {{Region}}

In the backend, when i edit a entity, if 1 or several "Region" options are checked, the "Dep" field displays the options available for the checked region. Good.
If no "Region" option are checked => Error : Too few argument to build the query string
If i try to edit the "DOCS" catalog in Catalog Manager : Same Error !

What i did in your Toolkit.php file

```
public static function prepareValueForQuery( $varValue ) {

        if ( !empty( $varValue ) && is_array( $varValue ) ) {

            // Custom
            if (empty($varValue[0])) {
	            return '';
            }
            // End custom
            $arrReturn = []; 
           …

```
May be not the good way to do this ?

**Comment by alnv** [comment]
This bug will be fixed with next update.

**Comment by alnv** [comment]
Is fixed in v1.29.2

**Comment by chibko** [comment]
Thanks Alexander.
